Migrated f-string codebase

sagnik1511 opened the pull request:
Updated  .\examples\python-guide\logistic_regression.py  file

### Changes made :
                           
                              print('Best `binary` time: ' + str(min(A)))  ---> print(f"Best `binary` time: {str(min(A))}")
                              print('Best `xentropy` time: ' + str(min(B))) ---> print(f"Best `xentropy` time: {str(min(B))}")

jameslamb reviewed (changes requested):
Thanks so much for this!

Please also change the following places:

* https://github.com/sagnik1511/LightGBM/blob/064e4868eca6fa893719dc50750d0a04cbc72e41/examples/python-guide/logistic_regression.py#L74
* https://github.com/sagnik1511/LightGBM/blob/064e4868eca6fa893719dc50750d0a04cbc72e41/examples/python-guide/logistic_regression.py#L84

<hr>

Some other general notes as well, which maintainers will appreciate in the future

* When you open a pull request, please reference any previous discussions in the description so that maintainers have enough context to understand the proposed changes. I've added "Contributes to #4136" to the PR description here.
* In the future, please use an informative title for your pull requests. This helps people to find it in the future from search, and makes the changelog for releases (e.g. https://github.com/microsoft/LightGBM/releases/tag/v3.2.1) easier to understand. "Migrated f-string codebase" is too broad. I've updated the title to be a bit more specific.
* In pull requests, it isn't necessary to explicitly write out what code changed. The PR description should just give a description of what you're proposing and why.

sagnik1511 commented:
> Thanks so much for this!
> 
> Please also change the following places:
> 
> * https://github.com/sagnik1511/LightGBM/blob/064e4868eca6fa893719dc50750d0a04cbc72e41/examples/python-guide/logistic_regression.py#L74
> * https://github.com/sagnik1511/LightGBM/blob/064e4868eca6fa893719dc50750d0a04cbc72e41/examples/python-guide/logistic_regression.py#L84
> 
> Some other general notes as well, which maintainers will appreciate in the future
> 
> * When you open a pull request, please reference any previous discussions in the description so that maintainers have enough context to understand the proposed changes. I've added "Contributes to #4136" to the PR description here.
> * In the future, please use an informative title for your pull requests. This helps people to find it in the future from search, and makes the changelog for releases (e.g. https://github.com/microsoft/LightGBM/releases/tag/v3.2.1) easier to understand. "Migrated f-string codebase" is too broad. I've updated the title to be a bit more specific.
> * In pull requests, it isn't necessary to explicitly write out what code changed. The PR description should just give a description of what you're proposing and why.

Thank you sir for your kind words. I've pulled a new request [here](https://github.com/microsoft/LightGBM/pull/4356). DO visit that :)
